### PR TITLE
Adding Squeenix & simpleMiniMap Square Map support

### PR DESCRIPTION
**Pull request by laytya**
subj

### CODE REVIEW

**Comment by Muehe** [comment]
:+1:

File: !Questie/astrolabe/Astrolabe.lua [hunk 1]
```diff
@@ -324,6 +324,9 @@ end
 local function placeIconOnMinimap( minimap, minimapZoom, mapWidth, mapHeight, icon, dist, xDist, yDist )
 	--TODO: add support for non-circular minimaps
 	local mapDiameter;
+	
+	
+
```
Comment (Muehe) [inline !Questie/astrolabe/Astrolabe.lua]: Why the newlines?

File: !Questie/astrolabe/Astrolabe.lua [hunk 2]
```diff
@@ -332,10 +335,23 @@ local function placeIconOnMinimap( minimap, minimapZoom, mapWidth, mapHeight, ic
 	local mapRadius = mapDiameter / 2;
 	local xScale = mapDiameter / mapWidth;
 	local yScale = mapDiameter / mapHeight;
-	local iconDiameter = ((icon:GetWidth() / 2) + 3) * xScale;
-
+	local iconDiameter = ((icon:GetWidth() / 2) -3) * xScale; -- LaYt +3
+
```
Comment (Muehe) [inline !Questie/astrolabe/Astrolabe.lua]: Remove tab.

**Comment by Schaka** [comment]
Basically clean up the empty spaces Muehe commented on and I'll merge it. 

Thank you.

**Comment by laytya** [comment]
Did it

**Comment by laytya** [comment]
any merge plans?

**Pull request merged**